catroid/src/main/java/org/catrobat/catroid/content/bricks/ShowTextColorSizeAlignmentBrick.java

```diff
@@ -56,12 +89,43 @@ public ShowTextColorAndSizeBrick(Formula xPosition, Formula yPosition, Formula s
 
 	@Override
 	public int getViewResource() {
-		return R.layout.brick_show_variable_text_color_and_size;
+		return R.layout.brick_show_variable_color_size_alignment;
 	}
 
 	@Override
 	protected int getSpinnerId() {
-		return R.id.show_variable_color_and_size_spinner;
+		return R.id.show_variable_color_size_spinner;
+	}
+
+	@Override
+	public View getView(Context context) {
+		super.getView(context);
+		List<Nameable> items = new ArrayList<>();
+		items.add(new AlignmentStyle(context.getString(R.string.brick_show_variable_aligned_left),
+				ALIGNMENT_STYLE_LEFT));
+		items.add(new AlignmentStyle(context.getString(R.string.brick_show_variable_aligned_centered),
+				ALIGNMENT_STYLE_CENTERED));
+		items.add(new AlignmentStyle(context.getString(R.string.brick_show_variable_aligned_right),
+				ALIGNMENT_STYLE_RIGHT));
+		BrickSpinner<AlignmentStyle> spinner =
+				new BrickSpinner<>(R.id.brick_show_variable_color_size_align_spinner, view, items);
+		spinner.setSelection(alignmentSelection);
+		spinner.setOnItemSelectedListener(new BrickSpinner.OnItemSelectedListener<AlignmentStyle> () {
```

Comment:
please remove the whitespace before `(` 

`<AlignmentStyle>()` instead of `<AlignmentStyle> ()`

[Checkstyle warning on Jenkins](https://jenkins.catrob.at/job/Catroid/job/PR-3106/19/checkstyleResult/)